error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/util/ThirdPartyLibrary.java

```diff
@@ -86,9 +89,15 @@ private static boolean isKnownClass(String className, VisitorState state) {
 
   private static boolean canLoadClass(String className, VisitorState state) {
     ClassFinder classFinder = ClassFinder.instance(state.context);
+    Symtab symtab = state.getSymtab();
+    // XXX: Drop support for targeting Java 8 once the oldest supported JDK drops such support.
+    ModuleSymbol module =
+        Source.instance(state.context).compareTo(Source.JDK9) < 0
+            ? symtab.noModule
+            : symtab.unnamedModule;
```

Comment:
Unconditionally using `noModule` _seems_ to work as well, but:
1. The [documentation](https://github.com/openjdk/jdk/blob/0ee196bef199c3d32c1f88b26eb4333a7ea73c10/src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symtab.java#L122-L124) claims it should only be used with `-source 8`.
2. I _suspect_ (but did not test) that if using `noModule` when targeting Java >9, the requested classes may be loaded a second time. That's not very nice/efficient.